generators/entity/index.js

```diff
@@ -1427,6 +1456,7 @@ module.exports = EntityGenerator.extend({
             this.fieldsContainBigDecimal = false;
             this.fieldsContainBlob = false;
             this.validation = false;
+            this.relationshipRequired = false;
```

Comment:
`relationshipRequired ` should at each relationship level so should be`relationship.relationshipRequired `